src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -485,12 +487,26 @@ def _confirm_watchdog_start(start_watchdog, cleanup_on_exit, fluent_connection):
     return start_watchdog
 
 
-def scm_to_py(topy, journal_file_names):
-    """Convert journal file names to Python file name."""
-    fluent_jou_arg = "".join([f'-i "{journal}" ' for journal in journal_file_names])
-    if isinstance(topy, str):
-        return f" {fluent_jou_arg} -topy={topy}"
-    return f" {fluent_jou_arg} -topy"
+@beartype
```

Comment:
This is used in other pyansys libraries, useful for runtime type-checking. The deleted TypeError is replaced by this.